docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
Change `FROM ubuntu:latest` to `FROM centos:6` and it's done. I don't think we should provide different versions of Context Broker for different SOs, as any Docker image should work on any Host SO without changes so, what do we have to gain having multiple images? Docker images should be black boxes of the applications, so there's no point, IMHO, in creating multiple flavours.